src/metrics.cpp

```diff
@@ -291,13 +300,23 @@ void ThreadShowMetricsScreen()
         lines += printMessageBox(cols);
         lines += printInitMessage();
 
-        // Explain how to exit
-        std::cout << "[" << _("Press Ctrl+C to exit") << "] [" << _("Set 'showmetrics=0' to hide") << "]" << std::endl;;
+        if (isScreen) {
+            // Explain how to exit
+            std::cout << "[" << _("Press Ctrl+C to exit") << "] [" << _("Set 'showmetrics=0' to hide") << "]" << std::endl;
+        } else {
+            // Print delineator
+            std::cout << "----------------" << std::endl;
```

Comment:
Perhaps make this longer, to match `To enable mining, add 'gen=1' to your zcash.conf and restart.`